jenkins/scripts/coverage/generate-index-html.py

```diff
@@ -72,8 +79,13 @@
   <div id="main">
     <div class="container">
       <h1>Node.js Nightly Code Coverage</h1>
+      Nightly combined JavaScript coverage, across Windows, Linux, and a variety of runtimes,
```

Comment:
It's not yet across "a variety of runtimes", is it? It's just Linux and Windows for now?

Reply:
removed the variety blurb for now.